numpy/core/einsumfunc.py

```diff
@@ -442,11 +442,11 @@ def _can_dot(inputs, result, idx_removed):
     --------
 
     # Standard GEMM operation
-    >>> _can_dot(['ij', 'jk'], 'ik', set('j'))
+    >>> _can_dot(['ij', 'jk'], 'ik', {'j'})
     True
 
     # Can use the standard BLAS, but requires odd data movement
-    >>> _can_dot(['ijj', 'jk'], 'ik', set('j'))
+    >>> _can_dot(['ijj', 'jk'], 'ik', {'j'})
```

Comment:
Better not to change this oen either, since as it was it matches the one on line 453